custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -300,3 +318,19 @@ def make_predictor_capabilities(supported_payload_formats):
             for payload_format, format_version in supported_payload_formats
         }
     }
+
+
+try:
+    import pyarrow
+except ImportError:
+    pyarrow = None
+
+
+def get_pyarrow_module():
```

Comment:
is this the best way to do this? I've never seen this pattern

Reply:
I've never seen the case like ours where we have optional module and part of the logic depending on it

I you've got better ideas - please let us know

Reply:
heres what I do https://github.com/datarobot/datarobot-user-models/blob/6100db14a27461c8ed06a48032dbbc04f345206d/custom_model_runner/datarobot_drum/drum/push.py#L43

Reply:
the idea was to have DRUM working if there's no Arrow, without forcing user to install it

Reply:
Although that is different I guess...

Reply:
I've seen Zohar using this, and learn from him:
https://github.com/datarobot/mlpiper/blob/c9af874c3ece97870887b3ef8b3ddde122281b7c/mlpiper-py/mlpiper/components/restful/metric.py#L8

I mean `try: import` is just standard way.
And how you pass imported module around, should not matter

Reply:
if user doesn't need Arrow - user won't notice it's missing

Reply:
> heres what I do
> 
> https://github.com/datarobot/datarobot-user-models/blob/6100db14a27461c8ed06a48032dbbc04f345206d/custom_model_runner/datarobot_drum/drum/push.py#L43

But you fail if module is not imported, here we don't.

Reply:
Here more opinions:
https://stackoverflow.com/questions/9689355/python-import-or-pass-modules-as-parameters

Reply:
So our way look fine to me.